tools/cmake/CheckPythonPackage.cmake

```
@@ -27,7 +27,9 @@ function(check_python_package PACKAGENAME OUTPUT_VAR)
         set(PY_INTERP_FOR_${OUTPUT_VAR} ${Python3_EXECUTABLE} CACHE INTERNAL "The python interpreter used to run the ${OUTPUT_VAR} check" FORCE)
 
         execute_process(
-            COMMAND ${Python3_EXECUTABLE} -c "import ${PACKAGENAME}" RESULT_VARIABLE PACKAGECHECK_RESULT
+            COMMAND ${Python3_EXECUTABLE} -c "import ${PACKAGENAME}"
+            RESULT_VARIABLE PACKAGECHECK_RESULT
+            ERROR_QUIET
```

Adding ERROR_QUIET to suppress the error message if the module is not found